addons/sale_crm/models/crm_lead.py

```diff
@@ -1,13 +1,13 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from openerp import models, fields, api, _, tools
 from datetime import datetime, date
 from dateutil.relativedelta import relativedelta
-import openerp.addons.decimal_precision as dp
+from openerp import models, fields, api, tools, _
```

Comment:
Import should be alphabetical order like : 
 from openerp import api, fields, models, tools, _